src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -140,6 +140,10 @@ public void runRightUp() {
         }
     }
 
+    public boolean getLiftEncoder() {
```

Comment:
The name of this function doesn’t really indicate what it does. isLiftAtTop would be more descriptive. Also we need a separate one for right and left